pennylane/shadows/transforms.py

```diff
@@ -42,7 +42,7 @@ def _replace_obs(tape: QuantumScript, obs, *args, **kwargs):
 
         # queue the new observable
         obs(*args, **kwargs)
-    qscript = qml.tape.QuantumScript.from_queue(q)
+    qscript = QuantumScript.from_queue(q)
```

Comment:
Replacing the actual qscript will be done later right? You could potentially already call them tape, but it could be confusing.

Reply:
will be done later. I want to keep using QuantumScript internally